.travis.yml

```diff
@@ -24,24 +24,13 @@ env:
     - secure: "MkxSVyjNwHPfuCA4kdrTeBRbmDV2j4wcvD7416ykZYSh2ixHnSFxDyi5cabCW2Rzz4+JDh2GUBL/mIJr5PvqKHRRV5Ehtu/fN8b53PjmcuREBzK3JMsuWtT0PMqfM633l0YAFhqLkoyDsRzpKofbw9LMQumapSOcHnODONoTgPfr0RixP7yaGkARB4ythdiCfbSK6BceS1q+XXfi2/aZh915l9Edjhs5T/3tJt8ufr8JuBV3dILk/3G7tgku6YI7SDNum6u6FVny+o23e+vzfkgDPDXVPECplAWaAt8VzasDafyVR1C4jFm+QY0D5QOnhcUmFZ04P1ng3690AI67QVgk+UUO3kpiFlKO4LhkKNf6z+gnGmYI87yz0VywcNrMwwiZC7f/yJee7mtF2YGndiO8NCISCORe3fi/gh19SOB4je8+g/CVWYU6tz7D5Ci5UKBMglnPP1o/TsO2J7VEbs9HqwfsBxghLo988KDeqtIaXCLY3Buro29jGgbYLcT9h7qG/Q3F1b4Z5Dga4FzPUQ4D11yXbv1pip1UNTwTWzyyY5YemIr0M7c4Z6JrndrrNact/iIUOW3rBoXVuWnltHfmVyXIfognBDGiB4YUHDdhSZd4etIQxID9cbEfdsL2xycrpH2bdqX9ePYK+DahBmx+qUESVpcMvW4cTYJygIY="
     # ENCRYPTED_IV
     - secure: "iz1NO2t2ZSEpc7CMz2CtMNXyFa4FTLRZu9wPlnx1IwXHTworqJ8ENSAv4kDHOFYWXubHb52c2c3dMMLLZtHxRpTtnHRvEDCHzLdc6RP5wnoOLogtfSzD140g94+iPEaXfg6hJ/Wv1nE+iBTtfDTpRMy8e/i+InU4af3FcwqMJW1oroslgd9ubZV+KUuv4nHZNOxTW67lLUQ2iRI3b7Ewjo/zkhMgCGLz4lnSO6KnspgKQ6VUA+UeV+Xx28Y5mDpH2ouhfajji6xUHptoB9AG2WZf6GfpOOMB5gxSnGNf6BiVal/TyG0unMiUy6dij+6XYnSpmG6EWna7nR35M4PD7pIwiOzcvwVwvZ4j5GnXIoIOA12Wd9vP8iPmji509slgsut00qFbgU4Uxii8ElGuDdChGzdVXuSYUBLtX7ldMcYKLMckyn/wBnWndQrDNv5jDROms+8YxNIFbm4kRM3ynt1xMFtm2v43Cj/VQ+ZwFV0cr3Y6y4QoThBrnuFXHIXJfe8HsuYZaGPpa9Jf1SDRNut7ClmvGm5+Yf+gkG1Csm82perZ1SQBtyBdtEPF71zQ1ktWYduQrG6jLb4On5zcGZ+CmPdjOopNatq6AatpFYpuZkVyhTbyn626fRIOCgbyDkgSS6yBweUB5eq8EW7Fdu8jqB0fR5+WKISe7m6dpTg="
-stages:
-  - prebuild
-  - build
 jobs:
   include:
-    - name: "Quality checks"
-      stage: "prebuild"
+    - name: "Lint"
```

Comment:
Perhaps `Quality checks` was more relevant given that we're not just doing a lint here